src/Concerns/DealsWithTenantSymlinks.php

```diff
@@ -27,7 +27,9 @@ protected static function possibleTenantSymlinks(Tenant $tenant): Collection
             $storagePath = str_replace('%storage_path%', $suffixBase . $tenantKey, $disks[$disk]);
             $publicPath = str_replace('%tenant_id%', $tenantKey, $publicPath);
 
-            $symlinks->push([public_path($publicPath) => storage_path($storagePath)]);
+            tenancy()->central(function() use ($symlinks, $publicPath, $storagePath) {
+                $symlinks->push([public_path($publicPath) => storage_path($storagePath)]);
+            });
```

Comment:
Reason: The paths are prefixed/suffixed manually (L27-28) because this action is intended to be used in the central context.  But when running the action as a tenant (which is possible at the moment), the path gets prefixed twice.

@stancl, should we keep it like this, or should we enforce not using the symlink actions in the tenant context? I think that since the actions accept collections of tenants, using the actions as a tenant doesn't make much sense – the tenant would be technically able to manage the symlinks of other tenants. But how to enforce the need to be in the central context to run these actions? Do we just check if tenancy is initialized, and if it is, throw an exception?

Reply:
Resolving this in response to Samuel's reply on BC